### PR TITLE
Migrated to docs.spring.io & updated sc-build

### PR DESCRIPTION
**Pull request by marcingrzejszczak**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
 
 = Spring Framework on Google Cloud Platform
 
-The Spring Cloud GCP project makes it easy for Spring users to run their applications on Google Cloud Platform.
-You can check our https://github.com/spring-cloud/spring-cloud-gcp/blob/master/README.adoc[project website] for more information.
+This project makes it easy for Spring users to run their applications on Google Cloud Platform.
+You can check our project website https://cloud.spring.io/spring-cloud-gcp[here].
 
-For a deep dive into the project, refer to the https://cloud.spring.io/spring-cloud-static/spring-cloud-gcp/1.2.2.RELEASE/reference/html/[Spring Cloud GCP 1.2 Reference Document] or the https://googleapis.dev/java/spring-cloud-gcp/latest/index.html[latest Javadocs].
+For a deep dive into the project, refer to the https://cloud.spring.io/spring-cloud-static/spring-cloud-gcp/1.2.0.RELEASE/reference/html/[Spring Cloud GCP 1.2 Reference Document].
```
Comment (meltsufin) [inline README.adoc:17]: This change to an older version doesn't seem right.
We probably need to update the source from which this is generated.

Comment (marcingrzejszczak) [reply README.adoc:17]: The new docs will be under https://docs.spring.io/spring-cloud-gcp/docs/1.2.2.RELEASE/ e.g. for 1.2.2.RELEASE version.

Comment (dzou) [reply README.adoc:17]: Hey @marcingrzejszczak, were the changes you made to this file manually made or did you generate the changes from sources somewhere?

I think we were just wondering why you changed the version from `1.2.3.RELEASE` to `1.2.0.RELEASE`.

Comment (dzou) [reply README.adoc:17]: Ah I see where the changes are coming from... I'll take care of merging this.

**Pull request merged**